plugin.video.nba/nbatvlive.py

```diff
@@ -62,70 +107,129 @@ def playLiveTV():
         log("live cookie: %s %s" % (querystring, livecookies), xbmc.LOGDEBUG)
 
         video_url = "http://%s/%s?%s|Cookie=%s" % (domain, arguments, querystring, livecookiesencoded)
-    else:
-        # Transform the link from adaptive://domain/url?querystring to
-        # http://domain/play?url=url&querystring
-        match = re.search('adaptive://([^/]+)(/[^?]+)\?(.+)$', video_adaptive_url)
-        domain = match.group(1)
-        path = urllib.quote_plus(str(match.group(2)))
-        querystring = match.group(3)
-        video_play_url = "http://%s/play?url=%s&%s" % (domain, path, querystring)
-        log("nba live tv: play url is %s" % video_play_url, xbmc.LOGDEBUG)
+        item = xbmcgui.ListItem(path=video_url)
+        xbmcplugin.setResolvedUrl(int(sys.argv[1]), True, item)
+
+    @staticmethod
+    def playLive():
+        if not vars.cookies:
```

Comment:
Sorry, I looked for the def and didn't see it. Looked like you were checking for fallout from the try preceding the check. Rest looks ok.